### PR TITLE
add VSA reviewed directories

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
- test codeowner behavior with small group of approvers 
- once this is merged we should validate 
  - does a codeowner github team need to have write access to the repo in order to be notified / approve the change 
  - only designated reviewers are notified 
  - do required reviewers appropriately understand their responsibility and how to manage the github team
- document process for creating and managing teams 

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,3 +4,8 @@
 # review when someone opens a pull request.
 
 *       @department-of-veterans-affairs/frontend-review-group
+
+# VSA
+src/applications/proxy-rewrite/redirects/crossDomainRedirects.json @department-of-veterans-affairs/vsa-frontend
```
Comment (rianfowler) [inline .github/CODEOWNERS]: i pulled these from recent changes made by kelson and nick. this one is very specific but I didn't want to assume VSA wanted to take full ownership of the proxy-rewrite component

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -4,3 +4,8 @@
...
+src/applications/find-va-forms @department-of-veterans-affairs/vsa-frontend
+src/platform/site-wide @department-of-veterans-affairs/vsa-frontend
```
Comment (rianfowler) [inline .github/CODEOWNERS]: I think this represents the bulk of what your team owns @ncksllvn . it includes the feature toggle stuff but I think that's fine for now. We aren't actively developing that and we can move that component into another platform folder.

**Review by rianfowler — COMMENTED** [review]
@ncksllvn @kelsonic let me know if you have any questions

File: .github/CODEOWNERS (hunk 2 above)
Comment (kelsonic) [reply .github/CODEOWNERS]: @rianfowler It would be awesome if we could get `src/applications/yellow-ribbon` too if that's possible! 🙂

Comment (kelsonic) [reply .github/CODEOWNERS]: @ncksllvn do you think there will be another app we maintain for the search epic?

Comment (rianfowler) [reply .github/CODEOWNERS]: We can add whatever apps you want as long as you have sole ownership of it.

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉Thanks for doing this @rianfowler!

**Review by jhonny218 — APPROVED** [review]

**Pull request merged**